src/app/resourcebundles/data/consumption/en.properties

```diff
@@ -2073,7 +2073,18 @@ messages.stmsg.m0143 = DOWNLOAD
 messages.stmsg.desktop.contentCantDownload = This item can’t be downloaded as of now
 frmelmnts.lbl.myEnrolledCollections = My Enrolled Collection
 frmelmnts.lbl.BatchExpiringIn = Batch expiring in
-
+frmelmnts.lbl.deleteConfirm = Are you sure you want to delete this Entity
+frmelmnts.lbl.createObserveAgain = Are you sure you want to create new instance
+frmelmnts.btn.addEntity = Add Entity
+frmelmnts.lbl.observationDetails = Observation Details
+frmelmnts.lbl.selectEntity = Select Entity
+frmelmnts.lbl.observeAgaian = Observe again
+frmelmnts.lbl.observationForm = Observation Form
+frmelmnts.btn.start = Start
+frmelmnts.btn.loadMore = Load More
+frmelmnts.lbl.deleteSubmission = Are you sure you want to delete instance
```

Comment:
question mark for all required texts

Reply:
But in design it was not there, will add for required texts.